scene.yaml

```diff
@@ -1,28 +1,27 @@
 ﻿# Author: Paitricio Gonzalez Vivo @patricio 2016
 import:
-    - http://tangrams.github.io/blocks/global.yaml
-    - http://tangrams.github.io/blocks/functions/zoom.yaml
-    - http://tangrams.github.io/blocks/functions/aastep.yaml
-    - http://tangrams.github.io/blocks/geometry/matrices.yaml
-    - http://tangrams.github.io/blocks/patterns/grid.yaml
-    - http://tangrams.github.io/blocks/shapes/cross.yaml
+    - https://tangrams.github.io/blocks/global.yaml
+    - https://tangrams.github.io/blocks/functions/zoom.yaml
+    - https://tangrams.github.io/blocks/functions/aastep.yaml
+    - https://tangrams.github.io/blocks/geometry/matrices.yaml
+    - https://tangrams.github.io/blocks/patterns/grid.yaml
+    - https://tangrams.github.io/blocks/shapes/cross.yaml
     - https://tangrams.github.io/blocks/elevation/stripes.yaml
     - https://tangrams.github.io/blocks/elevation/normal.yaml
     - https://tangrams.github.io/blocks/lines/dots.yaml
 
 sources:
     mapzen:
-        type: TopoJSON
-        url: https://tile.mapzen.com/mapzen/vector/v1/all/{z}/{x}/{y}.topojson
-        url_params:
-            api_key: mapzen-YoRQaZF
+        type: MVT
+        url: https://tile.nextzen.org/tilezen/vector/v1/512/all/{z}/{x}/{y}.mvt
         rasters: [normals-elevation]
         max_zoom: 16
+        url_params:
+            api_key: hx2J5tx6QDS1Pnhu8r97JQ
     normals-elevation:
         type: Raster
-        url: https://tile.mapzen.com/mapzen/terrain/v1/normal/{z}/{x}/{y}.png
+        url: https://tile.nextzen.org/tilezen/terrain/v1/normal/{z}/{x}/{y}.png?api_key=hx2J5tx6QDS1Pnhu8r97JQ
```

Comment:
Nit: It'd be better to use the `url_param: { api_key: * }` structure you had before instead of hard coding this, but it obviously works.

Reply:
oh right! addressed with new commit.